### ISSUE TITLE
Remove MCrypt encryption method

### ISSUE THREAD

**Issue by rlhawk** [d8-contrib-modules/encrypt#68]
It is generally recommended not to use MCrypt in modern web applications and the implementation that currently exists in Encrypt is not secure, so we should remove the MCrypt encryption method from the module. Tests are using this method, so we will need to add some sort of method to the tests. I will create a separate issue for that.

**Comment by svendecabooter** [comment]
https://github.com/d8-contrib-modules/encrypt/pull/83 removes the dependency on mcrypt in the test scenarios. Once that PR is merged, the MCrypt encryption method can probably be removed (maybe we should wait until a usable alternative in the form of the Defuse PHP Encryption library is available).

Same question as for PHPSecLib - should we move the code into a separate contrib module, even if we discourage using it?

**Comment by svendecabooter** [comment]
Mcrypt support has been removed now.
